cli/exchange/user.go

```diff
@@ -103,15 +120,9 @@ func UserSetHubAdmin(org, userPwCreds, user string, isHubAdmin bool) {
 	// Get the user from the exchange
 	var userList ExchangeUsers
 	httpCode := cliutils.ExchangeGet("Exchange", cliutils.GetExchangeUrl(), "orgs/root/users/"+userId, cliutils.OrgAndCreds(org, userPwCreds), []int{200, 404, 403}, &userList)
+
 	if httpCode == 200 {
-		// If the user already exists in the root org, we need to update it with a PUT to set hubAdmin to correct state
-		exchUser := cliutils.AddOrg("root", userId)
-		if existingUser, ok := userList.Users[exchUser]; !ok {
-			cliutils.Fatal(cliutils.NOT_FOUND, msgPrinter.Sprintf("user '%s' not found in list returned from exchange", exchUser))
-		} else {
-			updatedUser := cliutils.UserExchangeReq{Password: existingUser.Password, Admin: existingUser.Admin, HubAdmin: isHubAdmin, Email: existingUser.Email}
-			cliutils.ExchangePutPost("Exchange", http.MethodPut, cliutils.GetExchangeUrl(), "orgs/root/users/"+userId, cliutils.OrgAndCreds(org, userPwCreds), []int{201}, updatedUser, nil)
-		}
+		// Ignore this case if the user is already in root org
```

Comment:
If the user is in the root org, but wants to unset hubadmin, we need to give them an error message....but think about that for a moment....

I'm thinking that the ability to set hubadmin is no longer interesting. If users in the root org are always hubadmins, then why do we need to set and unset the hubadmin property on the user? A user in the root org cant unset hubadmin, and a user created in the root org is always a hubadmin, so that leaves creation and deletion of the user in the root org.